octoprint_filemanager/ThreadPool.py

```diff
@@ -1,33 +1,51 @@
-# http://code.activestate.com/recipes/577187-python-thread-pool/
```

Comment:
Can we readd the comment?

I really can't see any major rewrite that would justify removing the link to the original code.